### PR TITLE
Completed css-four-column-layout - Lisa Nguyen

### PR DESCRIPTION
**Pull request by lisa-t-nguyen**
image

### CODE REVIEW

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Figtree&family=Montserrat:wght@100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="howitworks">
+      <h1 class="howitworks">How It Works</h1>
+      <div class="fourphones">
+        <div class="firstphone">
```
Comment (Zhiwutian) [inline css-four-column-layout/index.html:14]: make sure each of the columns has a explicitly set width on it!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looks great Lisa, just make sure to set an explicit width on your columns!

**Comment by lisa-t-nguyen** [comment]
image
New screenshot with updated code

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**